judge/judgedaemon.main.php

```diff
@@ -975,12 +1005,14 @@ function disable(string $kind, string $idcolumn, $id, string $description,
 
 function read_metadata(string $filename): ?array
 {
-    if (!is_readable($filename)) return null;
+    if (!is_readable($filename)) {
+        return null;
+    }
```

Comment:
IMO this doesn't improve readability as now 2 more lines are spent on the same code, so 2 fewer lines fit on your screen.

Reply:
I personally really *do* like this (and similar) chnages. I don't like that one line does a check and an action...

Reply:
I really prefer always having braces `{}` for `if` statements.